skycoin/skywire
# get connected hypervisors

Did you run `make format && make check`? Yes

Fixes #1306 

 Changes:	
- add new command to `skywire-cli visor` as `chvpk` that return connected hypervisor PKs

Note:
we have `hvpk` that return all hypervisor PKs that available in config file, but this command return list of connected hypervisor PKs

How to test this PR:
- `make build`
- add two hypervisor PK to config, once available to connect and other be mock
- run visor
- check hvpk by `skywire-cli visor hvpk`
- check chvpk by `skywire-cli visor chvpk`
- should seems like:
  ```
  [mohammed@manjaro skywire]$ ./skywire-cli visor hvpk
  [0218905f5d9079bab0b62985a05bd162623b193e948e17e7b719133f2c60b92093 036c0e89735c675abce702a3d46ad1bf736bd8236dbd734bfaf368959de2c2b967]
  [mohammed@manjaro skywire]$ ./skywire-cli visor chvpk
  [036c0e89735c675abce702a3d46ad1bf736bd8236dbd734bfaf368959de2c2b967]
  ```
